cluster-bootstrap/src/main/resources/reference.conf

```diff
@@ -95,6 +95,12 @@ akka.management {
     # Configured how we communicate with the contact point once it is discovered
     contact-point {
 
+      # connect to the contact point by IP address instead of by hostname
+      #
+      # This is useful when coredns is configured with "pods disabled"
+      # Disabled by default because this is blocked when using Istio (#209)
+      connect-by-ip = no
```

Comment:
It might be my ignorance about k8s details, it's not clear to me from this description why it is needed.

So, with "pods disabled", the node cannot resolve names? Where does the IP address then come from?

On the other had, why doesn't it work with Istio? Does it add another layer of HTTP routing/reverse proxying that doesn't work without a proper `Host` header?

Reply:
Indeed when coredns is configured with `pods disabled`, no DNS names will be generated for individual pods. Nodes must then always go through other 'more high-level' names to find other  services. I'm not sure if they will use more abstract names and still get the direct IP's, or that they also use more high-level IP's and the routing will be done in the infrastructure. Perhaps both are possible.

Indeed Istio (if I understand correctly) inserts itself by making all pod DNS names resolve to the 'sidecar' and doing the routing there based on the `Host` headers.

This is relatively unfamiliar territory for me as well, though.